rasterio/plot.py

```diff
@@ -327,22 +330,18 @@ def show_hist(
         plt.show()
 
 
-def adjust_band(band, kind=None):
```

Comment:
we shouldn't delete methods, even if they are not used anymore here.

you can add a a deprecation warning if needed

Reply:
the adjust function has been restored in our scripts.